openff/interchange/components/interchange.py

```diff
@@ -56,7 +56,7 @@ class Interchange(DefaultModel):
     .. warning :: This API is experimental and subject to change.
     """
 
-    class InnerSystem(DefaultModel):
+    class _InnerSystem(DefaultModel):
```

Comment:
Good by me; this class is really only partially-implemented, but it's meant to be private-ish and should have probably have the underscore already